homeassistant/components/tessie/config_flow.py

```diff
@@ -0,0 +1,105 @@
+"""Config Flow for Tessie integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from http import HTTPStatus
+from typing import Any
+
+from aiohttp import ClientConnectionError, ClientResponseError
+from tessie_api import get_state_of_all_vehicles
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+TESSIE_SCHEMA = vol.Schema({vol.Required(CONF_ACCESS_TOKEN): str})
+
+
+class TessieConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config Tessie API connection."""
+
+    VERSION = 1
+    DOMAIN = DOMAIN
```

Comment:
Why are you setting domain as a class variable here? I don't see why this is needed

Reply:
I dont know either, I did it in my other integrations (which I copied to make this one), ill check if its required.